sql/moz-fx-data-shared-prod/amo_prod/desktop_addons_by_client_v1/metadata.yaml

```diff
@@ -17,6 +17,7 @@ bigquery:
     type: day
     field: submission_date
     require_partition_filter: false
+    expiration_days: 775
```

Comment:
I wonder if we should define this value of `755` in the `bqetl_project.yaml` config file instead to ensure that the tables where retention policy is set always have this value. This should allow us to lower the risk of an incorrect value being set.